packages/manager/apps/telecom/src/app/telecom/pack/move/building-details/move-building-details.controller.js

```diff
@@ -75,6 +77,25 @@ export default class MoveBuildingDetailsCtrl {
     this.$scope.$emit('updateBuildingDetails', this.selectedOffer);
   }
 
+  /* Check PTO reference value is valid */
+  isPtoRegexValid() {
+    this.ptoReferenceValid = this.model.ptoReference.match(
+      /^[A-Z]{2}-[A-Z\d]{4}-[A-Z\d]{4}$/,
+    );
+  }
+
+  /**
+   * Check if PTO reference is valid
+   * @returns boolean
+   */
+  isPtoValid() {
+    return (
```

Comment:
Maybe factorize this in some helpers (as I am reading twice the same code)

Reply:
yes I should think about it to do better code into a next feature.